### ISSUE TITLE
Add quickfix that swaps actual and expected value

### ISSUE THREAD

**Issue by inyutin-maxim** [meziantou/Meziantou.FluentAssertionsAnalyzers#1]
For example:
```csharp
"video1234_1234_test".Should().Be(result);
```
should be replaced with:
```csharp
result.Should().Be("video1234_1234_test");
```

**Comment by meziantou** [comment]
Good catch! What is the original assert code? Also, is the code using NUnit or xUnit?

**Comment by inyutin-maxim** [comment]
> Good catch! What is the original assert code? Also, is the code using NUnit or xUnit?

This code from my very old project, tests writen used NUnit.
Original Assert:

```csharp
Assert.AreEqual(result, "video1234_1234_test");
                ^^^^^^^, ^^^^^^^^
               expected, actual
```

but i think this problem can reproduce and for xUnit

**Comment by meziantou** [comment]
It seems you swapped the `expected` and `actual` parameters in the original call ([documentation](https://docs.nunit.org/articles/nunit/writing-tests/assertions/classic-assertions/Assert.AreEqual.html)).

The analyzer uses the following semantic:
````
Assert.AreEqual(expected, actual); // expected is the first parameter
actual.Should().Be(expected)
````

````
Assert.AreEqual(result, "video1234_1234_test");
// Should be
Assert.AreEqual("video1234_1234_test", result);
````

**Comment by inyutin-maxim** [comment]
> It seems you swapped the expected and actual parameters in the original call ([documentation](https://docs.nunit.org/articles/nunit/writing-tests/assertions/classic-assertions/Assert.AreEqual.html)).

I did this deliberately to show what the problem was, but in those distant times it was not at all obvious to me.
I started this problem to have a convenient tool for automatic replacement. I understand that a deeper semantic analysis is needed to smartly identify a potential error, and a warning need not always be shown, but this could make life easier. If you consider the problem redundant, you can close it

**Comment by inyutin-maxim** [comment]
I see an order like this:
1) I apply the fix for the original code: 
```csharp
Assert.AreEqual(result, "video1234_1234_test");
```
2) I get the following code:
```csharp
"video1234_1234_test".Should().Be(result);
```

3) I apply the next fix for the new code:
```csharp
"video1234_1234_test".Should().Be(result);
```
4) I get the following code:
```csharp
result.Should().Be("video1234_1234_test");
```

**Comment by meziantou** [comment]
I thought you were reporting a bug, but it's a feature request!

Something we can try is detecting constant values on the left side and report them because I don't think there is a valid reason to have a constant before "Should".

**Comment by inyutin-maxim** [comment]
yes you are right this new functionality, perhaps I did not exactly express myself when describing
